.pipelines/.vsts-vhd-builder-windows.yaml

```diff
@@ -97,4 +101,26 @@ stages:
           displayName: Setup Build Variables
         - template: ./templates/.builder-release-template-windows.yaml
           parameters:
-            artifactName: 2022-containerd
\ No newline at end of file
+            artifactName: 2022-containerd
+
+  - stage: build_vhd_2022_containerd_gen2
+    dependsOn: []
+    condition: eq('${{ parameters.build2022containerdgen2 }}', True)
+    jobs:
+    - job: build
+      timeoutInMinutes: 180
+      steps:
+        - bash: |
+            echo '##vso[task.setvariable variable=DRY_RUN]${{parameters.dryrun}}'
```

Comment:
Same question as that for [.pipelines/.vsts-vhd-builder-release-windows.yaml](https://github.com/Azure/AgentBaker/pull/2036/files#diff-7ff44dd52bb7628aab9652bdb37642f273e4ec43cd48b5e5f4c752bdba31478c)

Reply:
Added back https://github.com/Azure/AgentBaker/pull/2036/commits/7dfa348264f6453af6a2be809c1b517b66b94de8.